src/HierarchyAnalysisAlgorithm.cc

```diff
@@ -43,6 +43,7 @@ HierarchyAnalysisAlgorithm::HierarchyAnalysisAlgorithm() :
     m_endTimeLeafName{"event_end_t"},
     m_triggersLeafName{"triggers"},
     m_nhitsLeafName{"nhits"},
+    m_minHitsToSkip{2},
```

Comment:
This needs to be after `m_eventsToSkip{0}` to match the variable definition order in the header file and avoid a compilation error.